coverage_comment/coverage.py

```diff
@@ -89,7 +91,7 @@ def get_coverage_info(merge: bool, coverage_path: pathlib.Path) -> Coverage:
             )
         raise
 
-    return extract_info(data=json.loads(json_coverage), coverage_path=coverage_path)
+    return json_coverage, extract_info(data=json_coverage, coverage_path=coverage_path)
```

Comment:
I'm always hesitant to returning tuples. Sure, destructuring is great, but I'm always afraid of mixing up the order of elements. That said, risks are low here, so probably not worth the hassle of changing it.

Reply:
Yes, I wasn't entirely happy with that either, but with type annotations and CI we should also notice very quickly if we swap the two elements.